ord_schema/visualization/generate_text.py

```diff
@@ -69,38 +47,15 @@ def _generate(reaction, template_string, line_breaks):
 
 def generate_text(reaction):
     """Generates a textual reaction description."""
-    return _generate(reaction, _TEXT_TEMPLATE, False)
+    with open(os.path.join(os.path.dirname(__file__), 'template.txt'),
```

Comment:
Is there a reason you prefer this over defining a module-level constant for `_HTML_TEMPLATE` and `TEXT_TEMPLATE`? Requiring each call to `generate_text` or `generate_html` to read a file from disk seems somewhat inefficient.

Reply:
I guess I'm just generally avoidant of complex global initialization...these are only mostly constants since they can vary with whatever is in the file and that seems to rub against https://google.github.io/styleguide/pyguide.html#25-global-variables

I agree it's less efficient but I doubt it will noticeably affect performance. That said, I'll change it back if you feel strongly about this.

Reply:
I feel weakly about it, so we can keep it like you have